lib/identity_config.rb

```diff
@@ -0,0 +1,63 @@
+class IdentityConfig
+  class << self
+    attr_reader :store
+  end
+
+  CONVERTERS = {
+    uri: proc { |value| URI(value) },
+    string: proc { |value| value.to_s },
+    comma_separated_string_list: proc do |value|
+      value.split(',')
+    end,
+    integer: proc do |value|
+      Integer(value)
+    end,
+    json: proc do |value, options: {}|
+      JSON.parse(value, symbolize_names: options[:symbolize_names])
+    end,
+    boolean: proc do |value|
+      case value
+      when 'true'
+        true
+      when 'false'
+        false
```

Comment:
should we allow literal true/false too?

```suggestion
      when 'true', true
        true
      when 'false', false
        false
```

Reply:
👍🏼